### PR TITLE
Script for running modified main loop

### PR DESCRIPTION
**Pull request by VarunSatyadevShetty**

### CODE REVIEW

File: test_script_for_running_modified_main_loop.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import sys
+sys.path.append("/home/pi/Github_repo/preference_learning_dephy_exo")
```
Comment (unghee) [inline test_script_for_running_modified_main_loop.py]: nit: maybe add an instruction this should be changed based on persons' file structure?

File: test_script_for_running_modified_main_loop.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+import main_loop
```
Comment (unghee) [inline test_script_for_running_modified_main_loop.py]: does just importing make the file run?

**Review by unghee — CHANGES_REQUESTED** [review]
could we change the name to main_loop_optimization.py to be more clear? otherwise looks good!

File: test_script_for_running_modified_main_loop.py (hunk 2 above)
Comment (VarunSatyadevShetty) [reply test_script_for_running_modified_main_loop.py]: yes

**Pull request merged**